<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="my" name="com.wissenteil.oftest.test.RunCucumberTests" tests="2" failures="1" timestamp="13 Jun 2016 22:48:11 GMT" time="13.795" errors="0">
  <testcase name="feature" time="9.458" classname="com.wissenteil.oftest.test.RunCucumberTests"/>
  <testcase name="feature" time="4.337" classname="com.wissenteil.oftest.test.RunCucumberTests">
    <failure type="cucumber.runtime.CucumberException" message="org.junit.ComparisonFailure: expected:&lt;[0.88861]&gt; but was:&lt;[100]&gt;">
      <![CDATA[cucumber.runtime.CucumberException: org.junit.ComparisonFailure: expected:<[0.88861]> but was:<[100]>
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.junit.ComparisonFailure: expected:<[0.88861]> but was:<[100]>
	at org.junit.Assert.assertEquals(Assert.java:125)
	at org.junit.Assert.assertEquals(Assert.java:147)
	at com.wissenteil.oftest.stepDefinition.TestSteps.theExchangeRateWillbeSameAsOnFriday(TestSteps.java:50)
	at âœ½.Then the exchange rate will be same as on Friday(testFail.feature:9)
]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- com.wissenteil.oftest.test.RunCucumberTests -->
